wp-content/themes/core/theme.json

```diff
@@ -372,6 +372,12 @@
 		},
 		"blocks": {
 			"core/paragraph": {
+				"spacing": {
+					"margin": {
```

Comment:
Before we implement this, I'd prefer to get an updated Figma comp from design with the spacing redlined. There's an outstanding question to the design team in the [Figma file already](https://www.figma.com/file/zbjC33tZJz5A901jeXM7zv/Website-Design-System?type=design&node-id=1975-3703&mode=design&t=rhAkHHysAFWRe6Gj-0). We can track there.

Reply:
Core block spacing has been updated to match Figma.